pkg/tnf/interactive/spawner.go

```diff
@@ -146,6 +146,11 @@ type GoExpectSpawner struct {
 	verboseWriterIsSet bool
 	// verboseWriter is an alternate destination for verbose logs.
 	verboseWriter io.Writer
+
+	// sendTimeoutIsSet tracks whether the Send command timeout is set.
+	sendTimeoutIsSet bool
+	// sendTimeout is the timeout of send command
+	sendTimeout time.Duration
```

Comment:
I think the GetGoExpectOptions() function needs to be updated to include this option in the return array.

Reply:
Thanks for catching it.